features/admin.applications.v1/pages/applications-settings.tsx

```diff
@@ -0,0 +1,430 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { AlertLevels, IdentifiableComponentInterface } from "@wso2is/core/models";
+import { CopyInputField, DocumentationLink, EmphasizedSegment, PageLayout, useDocumentation } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Field, Form } from "@wso2is/form";
+import {
+    AppConstants,
+    history,
+} from "../../admin.core.v1";
+import { useGetDCRConfigurations } from "../api/use-get-dcr-configurations";
+import { updateDCRConfigurations } from "../api/applications-settings";
+import { AxiosError } from "axios";
+import { Dispatch } from "redux";
+import { useDispatch } from "react-redux";
+import { addAlert } from "@wso2is/core/store";
+import { getOIDCApplicationConfigurations } from "../api/application";
+
+/**
+ * Proptypes for the applications settings form component.
+ */
+interface ApplicationsSettingsPropsInterface extends IdentifiableComponentInterface {
+    /**
+     * Is the mandateSSA enabled.
```

Comment:
Let's add more descriptive doc comments. Some one new to codebase will not understand what SSA stands for :)

Reply:
Added with [d54ec35](https://github.com/wso2/identity-apps/pull/5964/commits/d54ec35c7ee212f1039c9afea0052b20884b03c3)